src/renderer/components/wallet/txs/TxForm.style.tsx

```diff
@@ -72,3 +72,38 @@ export const ButtonLinkIcon = styled(SelectOutlined)`
 export const Text = styled('span')`
   color: ${palette('text', 2)};
 `
+
+export const SuccessExtraContainer = styled.div`
```

Comment:
also make a container to have min 100% of possible height and align by vertical at the center

Reply:
Code pls ;)

Reply:
forget this as we desided to keep send forms similar

Reply:
I've updated the styles with latest commit c15b57d - pls check it again.